documentation/modules/ref-brokered-infra-config-example.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// assembly-infrastructure-configuration.adoc
+
+[id='ref-brokered-infra-config-example-{context}']
+= Brokered infrastructure configuration example
+
+The following example of a brokered infrastructure configuration file shows the various settings that can be specified.
+
+[source,yaml,options="nowrap",subs="attributes+"]
+----
+apiVersion: admin.enmasse.io/v1beta1
+kind: BrokeredInfraConfig
+metadata:
+  name: brokered-infra-config-example
+spec:
+  version: "{EnMasseVersion}" <1>
+  admin: <2>
+    resources:
+      memory: 256Mi
+    podTemplate:
+      metadata:
+        labels:
+          key: value
+  broker: <3>
+    resources:
+      memory: 2Gi
+      storage: 100Gi
+    addressFullPolicy: PAGE
+    podTemplate: <4>
+      spec:
+        priorityClassName: messaging
+----
+
+<1> Specifies the {ProductName} version used. When upgrading, {ProductName} uses
+this field to determine whether to upgrade the infrastructure to the requested version. If omitted,
+the version is assumed to be the same version as the controllers reading the configuration.
+
+<2> Specifies the settings you can configure for the `admin` components.
+
+<3> Specifies the settings you can configure for the `broker` components. Note that changing
+the `.broker.resources.storage` setting does not configure the existing broker storage size.
+
+<4> For both `admin` and `broker` components you can configure the following `podTemplate` elements:
+* `metadata.labels`
+* `spec.priorityClassName`
+* `spec.tolerations`
+* `spec.affinity`
+* `spec.containers.readinessProbe`
+* `spec.containers.livenessProbe`
+* `spec.containers.resources`
+* `spec.containers.env`
++
+All other `podTemplate` elements are ignored. For more information about these elements, see the {KubePlatform} documentation in the following Related links section.
```

Comment:
"Related links" should probably be italicized.

Also might want to consider adding a link to the section.